src/modules/status.js

```diff
@@ -0,0 +1,22 @@
+export default function updateTaskStatus() {
+  document.addEventListener('DOMContentLoaded', () => {
+    const inputChecks = document.querySelectorAll('.check');
+    inputChecks.forEach((check) => {
+      check.addEventListener('change', (e) => {
+        let array = [];
+        array = JSON.parse(localStorage.getItem('array') || '[]');
+        array.forEach((task) => {
+          if ((e.target.id === `input${task.index}`) && (task.completed === false)) {
+            task.completed = true;
+            e.target.checked = true;
+            localStorage.setItem('array', JSON.stringify(array));
+          } else if ((e.target.id === `input${task.index}`) && (task.completed === true)) {
+            task.completed = false;
+            e.target.checked = false;
+            localStorage.setItem('array', JSON.stringify(array));
+          }
+        });
+      });
+    });
+  });
+}
```

Comment:
- [ ] When I click on a task to mark it as completed, it does not happen in an instance, not until I refresh the page and then click completed again before it reflects in the localStorage. All todos I mark as completed reflects in the localStorage but when I refresh the browser, the changes maintain in the localStorage but the ticks on the check boxes for completed items do not show. Kindly watch the video below for a better demonstration and please try to fix it so it is interactive upon user interaction with the app. 👍🏽

[Interactive Requirement Video Demo](https://www.loom.com/share/ba794cdecbeb4a83a47c707eedd8bcc2)

Reply:
Addressed